CHANGES.rst

```diff
@@ -6,6 +6,7 @@ Changelog
 
 **Added**
 
+- Added 7 new Instruments
```

Comment:
This line is not explanatory about the changes included in this Pull Request. Please, add a line for each instrument with the number of Pull Request, like:
```
- #617 Instrument import interface: Shimadzu GCMS-QP2010 SE
- #617 Instrument import interface: Shimadzu GCMS-TQ8030 GC/MS/MS
- #617 Instrument import interface: Shimadzu ICPE-9000 Multitype
...
```
This way is easy later to find out which instruments are supported and since when, just by doing a quick search for "Instrument import interface".